zoo/services/tasks.py

```diff
@@ -34,6 +34,41 @@ def fetch_sentry_project_issues(project_slug: str) -> dict:
     return result
 
 
+def fetch_sonarqube_project_links(project_key: str) -> list:
+    api_request_url = "{0}/api/project_links/search?projectKey={1}".format(
+        settings.SONARQUBE_URL, project_key
+    )
+    auth = settings.SONARQUBE_TOKEN, ""
+
+    response = requests.get(api_request_url, auth=auth)
+    response.raise_for_status()
+
+    return response.json()["links"]
+
+
+def fetch_sonarqube_projects() -> list:
+    api_request_url = "{0}/api/projects/search".format(settings.SONARQUBE_URL)
```

Comment:
Same for this, and seeing that it's almost the same code but simply a different endpoint/params maybe we could abstract it to a function. Just to avoid the auth and the retrying part.